ydb/core/tx/datashard/export_s3_uploader.cpp

```diff
@@ -944,7 +944,10 @@ IActor* TS3Export::CreateUploader(const TActorId& dataShard, ui64 txId) const {
         Ydb::StatusIds::StatusCode status;
         TString error;
         FillTopicDescription(topic, pq.GetPersQueueGroup(), pq.GetSelf(), cdcStream.GetName(), status, error);
-
+        // Unnecessary fields
+        topic.Clearself();
+        topic.Cleartopic_stats();
```

Comment:
Лучше так (в одном стиле):
```suggestion
        topic.clear_self();
        topic.clear_topic_stats();
```